git_machete/client.py

```diff
@@ -1396,31 +1397,31 @@ def sync_annotations_to_github_prs(self) -> None:
 
     def __sync_annotations_to_definition_file(self, prs: List[GitHubPullRequest], current_user: Optional[str] = None) -> None:
         for pr in prs:
-            if pr.head in self.managed_branches:
+            if LocalBranch.of(pr.head) in self.managed_branches:
                 debug('sync_annotations_to_definition_file()',
                       f'{pr} corresponds to a managed branch')
                 anno: str = f'PR #{pr.number}'
                 if pr.user != current_user:
                     anno += f' ({pr.user})'
-                upstream: Optional[str] = self.up_branch.get(pr.head)
+                upstream: Optional[LocalBranch] = self.up_branch.get(LocalBranch.of(pr.head))
                 if pr.base != upstream:
                     warn(f'branch `{pr.head}` has a different base in PR #{pr.number} (`{pr.base}`) '
                          f'than in machete file (`{upstream or "<none, is a root>"}`)')
                     anno += f" WRONG PR BASE or MACHETE PARENT? PR has '{pr.base}'"
-                if self.__annotations.get(pr.head) != anno:
+                if self.__annotations.get(LocalBranch.of(pr.head)) != anno:
                     print(fmt(f'Annotating <b>{pr.head}</b> as `{anno}`'))
```

Comment:
While you're here:
```suggestion
                    print(fmt(f'Annotating `{pr.head}` as `{anno}`'))
```
(for more consistent formatting)